game/models.py

```
@@ -99,6 +99,81 @@ def save(self, *args, **kwargs):
+        default=0
+    )
+
+    draws = models.PositiveIntegerField(
+        default=0
+    )
+
+    updated_at = models.DateTimeField(
+        auto_now=True
+    )
+
+    def __str__(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)